src/datasources/queues/queue-consumer.service.interface.ts

```diff
@@ -0,0 +1,15 @@
+import { ConsumeMessage } from 'amqplib';
+
+export const IQueueConsumerService = Symbol('IQueueConsumerService');
+
+export interface IQueueConsumerService {
+  /**
+   * Checks if the configured queue consumer is connected.
+   */
+  isReady(): boolean;
+
+  subscribe(
+    queueName: string,
```

Comment:
Initially the `queueName` will be the name of the queue where the TXS is publishing the event messages, but this parameter opens the door to subscribing to different queues.